types/index.ts

```diff
@@ -38,7 +38,7 @@ export type Theme = {
 export interface User {
   id: string
   username: string
-  name: string
+  name: string | null
```

Comment:
instead of making the name ```null``` can you inspect your code base try to find what makes it null ?

Reply:
It was the user from google, however I added a fix.